static/js/click_handlers.js

```diff
@@ -337,13 +337,6 @@ export function initialize() {
             ui_util.blur_active_element();
         }
     });
-    $(".message_edit_form .send-status-close").on("click", function () {
```

Comment:
This is no longer needed because the `send_status` container doesn't need to be hidden and the rest is handled in the `send_status_close_button` click handler

Reply:
OK I notice that this has a `fadeOut` animation; I assume the other code path doesn't. I think we don't have a pattern of doing fadeOut when closing banners/popovers, so that's probably fine; after looking at the original UI, the `.stop(true)` part to send the progress bar to 0 before fading it out feels pretty weird anyway.